applications/asset_tracker_v2/src/modules/data_module.c

```diff
@@ -964,6 +964,12 @@ static void agps_request_handle(struct nrf_modem_gnss_agps_data_frame *incoming_
 {
 	int err;
 
+#if defined(CONFIG_NRF_CLOUD_AGPS) && defined(CONFIG_NRF_CLOUD_PGPS)
+	/* It is not necessary to request ephemerides and almanacs when P-GPS is enabled. */
+	incoming_request->sv_mask_ephe = 0;
```

Comment:
I think this needs to be removed before the PR can be merged, see the conversion in https://github.com/nrfconnect/sdk-nrf/pull/6152/files#r755855979.

Reply:
ok, what about atv1 changes?  revert those also?

Reply:
I think ATv1 supports simultaneous A-GPS and P-GPS use, at least there's an overlay for that. So that should be fine.